Source/JavaScriptCore/llint/LowLevelInterpreter.asm

```diff
@@ -406,6 +406,15 @@ macro dispatchOp(size, opcodeName)
     genericDispatchOpJS(dispatch, size, opcodeName)
 end
 
+macro superSamplerBegin(scratch)
+    leap _g_superSamplerCount, scratch
+    addi 1, [scratch]
```

Comment:
This is the same way to BaselineJIT / DFGJIT / FTLJIT.

Reply:
Do you think we should make it atomic? I guess we’d have to teach the LLInt how to emit those instructions then?

Reply:
Yes, but eventually.